autowarefoundation/autoware
# Feature/update autoware launcher

## Status
**REVIEW**

## Description
This is an update of Autoware Launcher (#1637).
Related PRs is autowarefoundation/autoware#1950

## Steps to Test or Reproduce
See Quick Start in [README.md](/CPFL/Autoware/blob/feature/update_autoware_launcher/ros/src/util/packages/autoware_launcher/README.md)
